### PR TITLE
chore: Remove `reply.getReponseTime()` 

### PR DESCRIPTION
**Pull request by ghost**
#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Review by gurgunday — COMMENTED** [review]
There is a bunch of other things to remove, not sure if we should do all the removals in one PR or not, because it will immediately increase conflicts if we need to backport to v4

**Comment by metcoder95** [comment]
Do you refer to `reply.getResponseTime` or overall all the deprecations?

**Review by mcollina — APPROVED** [review]
lgtm

**Review by gurgunday — APPROVED** [review]
I meant all deprecations in general, this seems good!

**Comment by mcollina** [comment]
@RafaelGSS can you take a look at why Nsolid is failing here?

**Review by metcoder95 — APPROVED** [review]

**Comment by RafaelGSS** [comment]
> @RafaelGSS can you take a look at why Nsolid is failing here?

Sure thing.

File: test/internals/reply.test.js [hunk 1]
```diff
@@ -49,7 +49,6 @@ test('Once called, Reply should return an object with methods', t => {
   t.equal(typeof reply.status, 'function')
   t.equal(typeof reply.header, 'function')
   t.equal(typeof reply.serialize, 'function')
-  t.equal(typeof reply.getResponseTime, 'function')
```
Comment (santigimeno) [inline test/internals/reply.test.js:52]: The `t.plan(16)` should be updated to `t.plan(15)` which I think causes the issues in the ci

**Comment by gurgunday** [comment]
Coverage still failing

**Comment by codershiba** [comment]
> Coverage still failing

How to fix?

**Review by gurgunday — COMMENTED** [review]
You just need to test these lines:

https://github.com/fastify/fastify/blob/25d829cfd6be8d7df65d84413d68a6309c104db6/lib/reply.js#L93-L95

**Comment by codershiba** [comment]
Still failing, not unable to view logs of test since they are so long :(

**Comment by climba03003** [comment]
> Still failing, not unable to view logs of test since they are so long :(

```
    not ok 50 - reply.elapsedTime should return 0 if the timer is not initialised on the reply by setting up response listeners # time=1.154ms
      ---
      failures:
        - tapError: incorrect number of tests
      ...
 ```

**Review by gurgunday — APPROVED** [review]
Looks great!

If you could chip away at the rest of the deprecations, it would be highly appreciated!

**Review by metcoder95 — APPROVED** [review]

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**